project_template/package.json

```diff
@@ -36,13 +36,12 @@
   },
   "devDependencies": {
     "autoprefixer": "^7.1.1",
-    "babel-core": "^6.24.1",
+    "@babel/core": "^7.0.0-beta.44",
```

Comment:
Unfortunately all of these beta modules are still not released.
They seem to be pretty stable, but for the time being I'll hold off creating a new Maji release.